src/kic/cmd/fleet/create.go

```
@@ -199,17 +200,23 @@ func validateClusterPrefix() bool {
 
 // run the command
 func runCreateCmd(cmd *cobra.Command, args []string) error {
-	// create the setup script from the template
-	createVMSetupScript()
-
 	if dryRun {
 		return doDryRun()
 	}
 
-	if digitalOcean {
-		// add the GitOps target
-		addTarget(cluster, ssl)
+	// add the GitOps target
```

this was moved to the top to "fail fast" to resolve issues we've seen in onboarding